test/UserService.Data.UnitTests/UserCredentialsRepositoryTests.cs

```diff
@@ -17,10 +17,10 @@ class UserCredentialsRepositoryTests
         private IDataProvider provider;
         private IUserCredentialsRepository repository;
 
-        private DbUserCredentials firstUserCredentials;
-        private DbUserCredentials secondUserCredentials;
-        private Guid firstUserId;
-        private Guid secondUserId;
+        private Guid userId;
+        private string salt;
+        private DbUserCredentials userCredentials;
+        internal const string SALT3 = "LT.DigitalOffice.SALT3";
```

Comment:
Remove this one from here, use 
https://docs.microsoft.com/ru-ru/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=netcore-3.1